repairnator/docker-images/kubernetes/pipeline-dockerimage/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM maven:3.6.1-jdk-8
```

Comment:
How is this docker-file different to the current pipeline image? (I see the stomp installation, but why would this be necessary?).

Reply:
Stomp is for activeMQ without it there will be no queuing :) .

Reply:
So you keep a queue inside of the pipeline? Wouldn't that potentially mess with state in the container?

Reply:
Ah no, the stomp is required to communicate with the global queue on the queue server. The queue is not in the pipeline but on activeMQ server.

Reply:
Okay, so it is just to say "Hey, I'm done and ready for my next job"?

Reply:
Yes , basicly it is more desirable that way since we don't need to think about which job to assign to whom :).